variants/deploy_with_ackama_ec2_capistrano/template.rb

```diff
@@ -0,0 +1,235 @@
+puts "=" * 80
+puts "Starting variant: deploy_with_ackama_ec2_capistrano"
+puts "=" * 80
+
+copy_file "variants/deploy_with_ackama_ec2_capistrano/ackama_aws.yml", "ackama_aws.yml"
```

Comment:
This file name could be better I think

Reply:
I'm open to suggestions - I deliberately made sure the gem was generic to avoid making it unnecessarily vendor-lock-in-y, so think we should be fine with `aws.yml` which matches other stuff in `config/` like `database.yml`, `webpacker.yml`, etc.

(The naming has been the stiffest of things about this gem, as I wanted to avoid being overly verbose or specific given it could be doing more than `ec2`, but also it's not _creating_ anything which could be a bit confusing... 😅)

Reply:
I like the idea of putting it in `config/`. Thinking about this more, I think it should match the gem name (it's essentially a config file for the gem) so `config/aws_ec2_environment.yml`

Reply:
What about `config/aws_ec2_environments.yml`, since it defines multiple? (Whereas the gem represents a single one)

Reply:
The existing convention seems to be singluar names e.g. `config/database.yml` sets up all database envs, `config/storage.yml` does similar for the various storage envs so I think I prefer the singluar. :bike: 🛖

Reply:
Yup in that case singular sounds fine